monkey/monkey_island/cc/ui/src/components/reactive-graph/ReactiveGraph.tsx

```diff
@@ -8,7 +8,12 @@ const GraphWrapper = (props: { graph: Graph, events: any }) => {
   let options = getOptions();
   return (
     <div className={'net-graph-wrapper'}>
-      <Graph graph={props.graph} options={options} events={props.events}/>
+      <Graph graph={props.graph} options={options} events={props.events}
+      getNetwork={network => {
+        // Move the center of the camera to the right
+        // to compensate for the side panel on the right
+        network.moveTo({'position': {'x': 130, 'y': 0}})
```

Comment:
Is there a constant we can define somewhere rather than using a magic number here? Both the side panel and this component should rely on it.